dev/ruff.py

```diff
@@ -27,10 +19,7 @@ def transform(stdout: str, is_maintainer: bool) -> str:
                 )
                 line = f"{line}. Run {command} to fix this error."
             else:
-                name = get_rule_name(m.group(1))
-                line = (
-                    f"{line}. See https://beta.ruff.rs/docs/rules/{name} for how to fix this error."
-                )
+                line = f"{line}. See https://docs.astral.sh/ruff/rules/{m.group(1)} for how to fix this error."
```

Comment:
The ruff documentation now redirects `https://docs.astral.sh/ruff/rules/{error_code}` to `https://docs.astral.sh/ruff/rules/{error_name}` (example: https://docs.astral.sh/ruff/rules/F401/). code -> name conversion is no longer needed.